src/blockchain/bulk-sync-augur-node-with-blockchain.ts

```diff
@@ -24,9 +24,16 @@ function setSyncFinished() {
   augurEmitter.emit(SubscriptionEventNames.SyncFinished);
 }
 
-export async function bulkSyncAugurNodeWithBlockchain(db: Knex, augur: Augur, blocksPerChunk: number|undefined): Promise<number> {
-  const row: HighestBlockNumberRow|null = await db("blocks").max("blockNumber as highestBlockNumber").first();
-  const lastSyncBlockNumber: number|null|undefined = row!.highestBlockNumber;
+export async function bulkSyncAugurNodeWithBlockchain(db: Knex, augur: Augur, blocksPerChunk: number | undefined): Promise<number> {
+  while (true) {
```

Comment:
Looks good. This might be a useful function to have broken out, I can imagine a few different places you might want to await for sync.

Reply:
Great idea. Fixed next push